### ISSUE TITLE
Bat theme not picked up from environment

### ISSUE THREAD

**Issue by kristijanhusak** [nvim-telescope/telescope.nvim#488]
### Description
Bat theme not applied to the preview window. Gif at the end of description.

**Expected Behavior**
Expect telescope to read `$BAT_THEME` env variable

**Actual Behavior**
Looks like it uses the default

### Steps to reproduce
1. nvim -nu test.vim
2. :Telescope find_files
3. Find some file
4. Check preview
5. Exit neovim, open same file in bat from cli, see difference

### Environment
* nvim --version output: VIM v0.5.0-dev+1062-gcc1851c9f
* Operating system: Manjaro Linux 
* Telescope commit: 9e76b16

### Min init.vim

```viml
set nocompatible hidden laststatus=2
set rtp+=/home/kristijan/.config/nvim/pack/packager/start/popup.nvim
set rtp+=/home/kristijan/.config/nvim/pack/packager/start/plenary.nvim
set rtp+=/home/kristijan/.config/nvim/pack/packager/start/telescope.nvim

lua << EOF
require('telescope').setup{
  defaults = {
    vimgrep_arguments = {
      'rg',
      '--color=never',
      '--no-heading',
      '--with-filename',
      '--line-number',
      '--column',
      '--smart-case'
    },
    prompt_position = "bottom",
    prompt_prefix = ">",
    selection_strategy = "reset",
    sorting_strategy = "descending",
    layout_strategy = "horizontal",
    layout_defaults = {},
    file_ignore_patterns = {},
    shorten_path = true,
    winblend = 0,
    width = 0.75,
    preview_cutoff = 120,
    results_height = 1,
    results_width = 0.8,
    border = {},
    borderchars = { '─', '│', '─', '│', '╭', '╮', '╯', '╰'},
    color_devicons = true,
    use_less = true,
  }
}
EOF

```

Gif:
![telescope-bat](https://user-images.githubusercontent.com/1782860/106575242-25b51680-653c-11eb-890f-dfb5ac432056.gif)

**Comment by Conni2461** [comment]
If think most of us moved to `buffer_previewers` in the last few months, at least i did, so its hard for me to notice this issues 😆 

But from code perspective everything should be fine.
There is this function:
https://github.com/nvim-telescope/telescope.nvim/blob/9e76b1613a25cc6f63d17eb73432523924662619/lua/telescope/previewers/term_previewer.lua#L35-L66

We get the theme at 41 and apply at in line 60. Maybe you wanna debug it yourself. If not i will take a look at it this evening :)

Did it work like 3 days ago before the env was fixed upstream?

**Comment by Conni2461** [comment]
Okay i just check it. We no longer need to do that. Feel free to make a PR :)

**Comment by kristijanhusak** [comment]
Great :)

I wasn't even aware of the buffer previewers. That works perfectly. Why isn't it the default?

**Comment by Conni2461** [comment]
#420 i will make them default very soon. Just trying to finish some more things.

We decided to make them optional when i shipped them #298 last year because often we would get no or the wrong highlighting. The vimgrep/qflist previewer weren't async etc ... Fixed many of those issues and i am almost done with what i had in mind for previewers and telescope 1.0 :)
